ImagePickerSheetController/ImagePickerSheetController/ImagePickerSheetController.swift

```diff
@@ -429,13 +429,8 @@ extension ImagePickerSheetController: UICollectionViewDelegate {
         if let maximumSelection = maximumSelection {
             if selectedAssetIndices.count >= maximumSelection,
                 let previousItemIndex = selectedAssetIndices.first {
-                    let deselectedAsset = selectedAssets[previousItemIndex]
-                    delegate?.controller?(self, willDeselectAsset: deselectedAsset)
```

Comment:
Why is it unsafe/undesired to call the delegate methods here?  Line 433 is, effectively, deselecting the first selected item, and so notifying the delegate makes sense to me.